openmetadata-spec/src/main/resources/json/schema/entity/services/connections/objectstore/s3ObjectStoreConnection.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/store/s3StoreConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "S3 StoreConnection",
+  "description": "S3 Store Connection.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.services.connections.objectstore.S3Connection",
+  "properties": {
+    "awsConfig": {
```

Comment:
we need to add the `type` property to handle `pydantic` conversion when parsing the dictionary. For example: https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/entity/services/connections/database/mysqlConnection.json#L23

Reply:
this will apply to S3, GCS and Azure services